### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by msmichellegar**
Adding CircleCI etc.

### CODE REVIEW

**Comment by msmichellegar** [comment]
@zapthedingbat Terraformised here: https://github.com/conde-nast-international/cnid-infrastructure/pull/114

Think this is ready for final review!

File: README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+An Amazon Lambda that receive alarm notifications from CloudWatch and posts to Slack. Based on [this](https://github.com/ashiina/aws-lambda-cloudwatch-slack). This listens to an SNS topic called `alarms-production` (or `alarms-staging`), to which Cloudwatch notifications are posted.
 
 ## Environment Variables
 
 The following environment variables need to be set:
 
 | name | description |
 |:-----------|:------------|
-| SLACK_HOOK | The hook URL for your Slack [Incoming Webhook integration](https://api.slack.com/incoming-webhooks). Something like `https://hooks.slack.com/{path}`. |
+| COCO_ALARMS_SLACK_HOOK | The hook URL for your Slack [Incoming Webhook integration](https://api.slack.com/incoming-webhooks). Something like `https://hooks.slack.com/{path}`. |
```
Comment (gazs) [inline README.md]: if this is a general 'cloudwatch slack alarm', would a more generic name be better?

Comment (msmichellegar) [inline README.md]: Yep, agree. Was tossing up but decided I am going to do that in a different PR as I have to change quite a few things to generalise the Lambda. This is the current environment variable in `cnid_infrastructure`.

Comment (gazs) [inline README.md]: can you change the infrastructure first before merging this? i think that would be cleaner

Comment (msmichellegar) [inline README.md]: https://github.com/conde-nast-international/cnid-infrastructure/pull/122 @gazs

**Comment by gazs** [comment]
👍 👍 👍
